db/institutions.yml

```diff
@@ -1492,3 +1492,252 @@ zhejiangu:
   updated: 2020-02-14 19:37:28.925621
   uuid: 4f88b3c9-4719-4a0e-836b-16af6ab6605f
   year: 2020
+lhmfcas:
+  aka:
+    - Laboratory of High Magnetic Field of Chinese Academy of Sciences
+    - lhmfcas
+  city: Hefei
+  country: China
+  day: 15
+  month: 3
+  name: Laboratory of High Magnetic Field of Chinese Academy of Sciences
+  year: 2020
+lab49:
+  aka:
+    - lab49
+  city: Dublin
+  country: Ireland
+  day: 15
+  month: 3
+  name: Lab49
+  year: 2020
+kyotou:
+  aka:
+    - kyotou
+  city: Kyoto
+  country: Japan
+  day: 15
+  month: 3
+  name: Kyoto University
+  year: 2020
+kenyattau:
+  aka:
+    - kenyattau
+  city: Kahawa
+  country: Kenya
+  day: 15
+  month: 3
+  name: Kenyatta University
+  year: 2020
+icmm:
+  aka:
+    - Instituto de Ciencia de Materiales de Madrid
+    - icmm
+  city: Madrid
+  country: Spain
+  day: 15
+  month: 3
+  name: Instituto de Ciencia de Materiales de Madrid
+  year: 2020
+ideas:
+  aka:
+    - ideas
+    - Institute for the Development of Energy for African Sustainability
+  city: Pretoria
+  country: South Africa
+  day: 15
+  month: 3
+  name: Instituto de Ciencia de Materiales de Madrid
+  year: 2020
+icl:
+  aka:
+    - icl
+    - Imperial College of London
+  city: London
+  country: England
+  day: 15
+  month: 3
+  name: Instituto de Ciencia de Materiales de Madrid
+  year: 2020
+huazhongu:
+  aka:
+    - huazhongu
+  city: Wuhan
+  country: China
+  day: 15
+  month: 3
+  name: Huazhong University
+  year: 2020
+huawei:
+  aka:
+    - Huawei
+  city: Shenzhen
+  country: China
+  day: 15
+  month: 3
+  name: Huawei
+  year: 2020
+hermes:
+  aka:
+    - hermesm
+    - hermes
+    - hermesmicrovision
+  city: Veldhoven
+  country: The Netherlands
+  day: 15
+  month: 3
+  name: Hermes Microvision
+  year: 2020
+hmcollege:
+  aka:
+    - harveymuddcollege
+    - hmcollege
+  city: Claremont
+  country: USA
+  day: 15
+  month: 3
+  name: Harvey Mudd College
+  state: CA
+  year: 2020
+formlabs:
+  aka:
+    - Formlabs
+    - formlabs
+  city: Somerville
+  country: USA
+  day: 15
+  month: 3
+  name: Formlabs
+  state: MA
+  year: 2020
+ebi:
+  aka:
+    - ebi
+  city: Hinxton
+  country: England
+  day: 15
+  month: 3
+  name: European Bioinformatics Institute
+  year: 2020
+dscsolutions:
+  aka:
+    - dscsolutions
+  city: Leola
+  country: USA
+  day: 15
+  month: 3
+  name: DSC Solutions
+  state: PA
+  year: 2020
+desy:
+  aka:
+    - desy
+  city: Hamburg
+  country: Germany
+  day: 15
+  month: 3
+  name: German Electron Synchroton
```

Comment:
Keep this in aka, but here maybe  Deutsche Elecrtron Synchrotron (or whatever is the correct spelling)